policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -383,8 +383,7 @@ impl kubert::index::IndexNamespacedResource<linkerd_k8s_api::EgressNetwork> for
         self.resource_info
             .insert(egress_net_ref, egress_network_info);
 
-        self.reindex_resources();
-        self.reinitialize_egress_watches();
+        self.reinitialize_egress_watches(self.egress_net_target_ns(Arc::new(ns)));
```

Comment:
Is it intentional the reindex_resources got removed here?

Reply:
No, this was a mistake. Nice catch.